### PR TITLE
Fix run SDL on QNX 6.6

### PR DESCRIPTION
**Pull request by bonewell**
- Change config for QDB

@VladSemenyuk @DKlimenko

### CODE REVIEW

**Comment by VladSemenyuk** [comment]
reviewed

**Comment by VladSemenyuk** [comment]
Reviewed

File: src/components/ivdcm_module/CMakeLists.txt [hunk 1]
```diff
@@ -37,7 +37,10 @@ set (SOURCES
 )
 
 set (LIBRARIES
+    jsoncpp
+    ApplicationManager
```
Comment (DKlimenko) [inline src/components/ivdcm_module/CMakeLists.txt]: Not sure if we need it

**Comment by DKlimenko** [comment]
Reviewed

File: src/components/ivdcm_module/CMakeLists.txt (hunk 1 above)
Comment (VladSemenyuk) [inline src/components/ivdcm_module/CMakeLists.txt]: We need it for core_service

File: src/appMain/init_policy.sh [hunk 2]
```diff
@@ -1,6 +1,24 @@
-# This script star QDB server for SDL
+# This ksh script starts QDB server for SDL on QNX
 # Need superuser to start qdb
 
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/qnx650/target/qnx6/x86/usr/lib
-export LD_LIBRARY_PATH
-/usr/sbin/qdb -c policy.ini
+version=`uname -r`
+echo "QNX ${version}"
```
Comment (pestOO) [inline src/appMain/init_policy.sh]: @bonewell, looks liek it shall be build only on QNX system?
Whats about cross-building?

**Pull request merged**